test/system/helpers.bash

```diff
@@ -1,5 +1,11 @@
 # -*- bash -*-
 
+# Ignore cgroups-v1 warning in cirrus
+if [[ ! -z "$CIRRUS_CI" && $(mount | grep -q cgroup2) -ne 0 ]]; then
```

Comment:
Suggestion: `-n`, not bang-dash-z. 

Suggestion: `is_cgroupsv1`, not the `mount-grep` thing.

Actually, taking a step back, just remove this completely and set it in `.cirrus.yml` and in the RHEL `gating-whatever.yml`.

Re: debian, as @Luap99 says we use debian-in-CI as a proxy for RHEL (cgroups v1, runc). If we switch Debian to cgroups v2, we probably want to switch to crun also.